stdlib/public/core/LifetimeManager.swift

```diff
@@ -73,18 +77,20 @@ public func _fixLifetime<T>(_ x: T) {
 ///     The pointer argument is valid only for the duration of the function's
 ///     execution.
 /// - Returns: The return value, if any, of the `body` closure.
-@inlinable
 @_alwaysEmitIntoClient
-public func withUnsafeMutablePointer<T, E: Error, Result>(
+public func withUnsafeMutablePointer<
+  T: ~Copyable, E: Error, Result: ~Copyable
+>(
   to value: inout T,
   _ body: (UnsafeMutablePointer<T>) throws(E) -> Result
 ) throws(E) -> Result {
   try body(UnsafeMutablePointer<T>(Builtin.addressof(&value)))
 }
 
+@_spi(SwiftStdlibLegacyABI) @available(swift, obsoleted: 2)
```

Comment:
Did we ever figure out if we could use 1 here? Also, I'd prefer these attributes be on different lines, but it's not a show stopper considering we shouldn't touch these things anymore.

Reply:
Also, can we dump these into the `LegacyABI.swift` to clean up where actual public API is? (Suggestion, not a show stopper)

Reply:
We can do this cleanup in another PR.

Reply:
`@available(swift, obsoleted: 0)` does not work, but `@available(swift, obsoleted: 1)` does, so I'm switching to that! 👍 

Dumping these in a file we never look at is indeed a todo item for next year or so. For now, I think we want to keep the original definitions right beside the new versions.